vocabularies/functions.ttl

```diff
@@ -420,13 +420,37 @@ geofp:transform_param1 a fno:Parameter ;
 
 geofp:transform_param2 a fno:Parameter ;
     fno:type xsd:anyURI ;
-    fno:required "true"^^xsd:boolean ;
+    fno:required "false"^^xsd:boolean ;
```

Comment:
This should be true, correct? Since we decided to use a named function instead of a default parameter value.

Reply:
Thx. It is now corrected.